### PR TITLE
Update PyTorch/XLA r2.5 benchmarking with rc10 wheel

### PR DESCRIPTION
**Pull request by ManfeiBai**
# Description

Please include a summary of relevant context/issue and your changes.

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**Instruction and/or command lines to reproduce your tests:** ...

**List links for your tests (use go/shortn-gen for any internal link):** ...

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: dags/pytorch_xla/configs/pytorchxla_torchbench_config.py [hunk 1]
```diff
@@ -85,16 +85,16 @@ class R2_4(enum.Enum):
     TORCH_XLA_REPO_BRANCH = "-b v2.4.0-rc8"
 
   class R2_5(enum.Enum):
-    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0rc9-cp310-cp310-linux_x86_64.whl"
-    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0rc9-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl"
     TORCH = "torch==2.5.0"
     TORCHVISION = "torchvision==0.19.0"
```
Comment (zpcore) [inline dags/pytorch_xla/configs/pytorchxla_torchbench_config.py:91]: Note that the torchvision and torchaudio should be compatible with the torch. The final release test is not running in airflow at this time.

**Review by zpcore — APPROVED** [review]
LGTM!

File: dags/pytorch_xla/configs/pytorchxla_torchbench_config.py (hunk 1 above)
Comment (ManfeiBai) [reply dags/pytorch_xla/configs/pytorchxla_torchbench_config.py:91]: thanks, do we know how to confirm the compatible torchvision and torchaudio now?

Comment (ManfeiBai) [reply dags/pytorch_xla/configs/pytorchxla_torchbench_config.py:91]: torchvision looks like preparing 0.20 release too, let me confirm locally which torchvision does PyTorch 2.5.0 is using now with `pip3 install torch==2.5.0 torchvision --index-url https://download.pytorch.org/whl/test/cpu`

Comment (ManfeiBai) [reply dags/pytorch_xla/configs/pytorchxla_torchbench_config.py:91]: ok, it installed torchvision 0.20:
```
(base) manfei@t1v-n-136f5173-w-0:~$ pip3 install torch==2.5.0 torchvision --index-url https://download.pytorch.org/whl/test/cpu
Looking in indexes: https://download.pytorch.org/whl/test/cpu
Collecting torch==2.5.0
  Downloading https://download.pytorch.org/whl/test/cpu/torch-2.5.0%2Bcpu-cp312-cp312-linux_x86_64.whl (174.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 174.6/174.6 MB 61.0 MB/s eta 0:00:00
Collecting torchvision
  Downloading https://download.pytorch.org/whl/test/cpu/torchvision-0.20.0%2Bcpu-cp312-cp312-linux_x86_64.whl (1.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 MB 24.9 MB/s eta 0:00:00
Collecting filelock (from torch==2.5.0)
  Downloading https://download.pytorch.org/whl/test/filelock-3.13.1-py3-none-any.whl (11 kB)
Collecting typing-extensions>=4.8.0 (from torch==2.5.0)
  Downloading https://download.pytorch.org/whl/test/typing_extensions-4.12.2-py3-none-any.whl (37 kB)
Collecting networkx (from torch==2.5.0)
```

Comment (ManfeiBai) [reply dags/pytorch_xla/configs/pytorchxla_torchbench_config.py:91]: install torchaudio 2.5.0 too:
```
(base) manfei@xxx:~$ pip3 install torch==2.5.0 torchvision torchaudio --index-url https://download.pytorch.org/whl/test/cpu
Looking in indexes: https://download.pytorch.org/whl/test/cpu
Collecting torch==2.5.0
  Using cached https://download.pytorch.org/whl/test/cpu/torch-2.5.0%2Bcpu-cp312-cp312-linux_x86_64.whl (174.6 MB)
Collecting torchvision
  Using cached https://download.pytorch.org/whl/test/cpu/torchvision-0.20.0%2Bcpu-cp312-cp312-linux_x86_64.whl (1.8 MB)
Collecting torchaudio
  Downloading https://download.pytorch.org/whl/test/cpu/torchaudio-2.5.0%2Bcpu-cp312-cp312-linux_x86_64.whl (1.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.7/1.7 MB 20.2 MB/s eta 0:00:00
Requirement already satisfied: filelock in ./miniconda3/lib/python3.12/site-packages (from torch==2.5.0) (3.13.1)
```

Comment (ManfeiBai) [reply dags/pytorch_xla/configs/pytorchxla_torchbench_config.py]: let's test with this compatible version now

**Pull request merged**